src/lib/TranscriptEditor/MediaPlayer/RollBack.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './RollBack.module.css';
+class RollBack extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+    render() {
+        return (
+          <div>
+            <p className={ styles.helpText }>Rollback
+              <b> <output >{ `x${ this.props.rollBackValueInSeconds }` }</output></b> Seconds
```

Comment:
Same with here, can adjust font-weight in the css rather than adding lots of wrapper tags. Again, no biggie but since there's a css file for the Rollback component, let's use it 🙌

Reply:
ok, adding to style and removing `b`